### PR TITLE
List cluster, monitoring integration and import cluster workflow

### PR DESCRIPTION
**Pull request by anivargi**
Cluster List with monitoring stats
Node List with monitoring stats

### CODE REVIEW

**Comment by anivargi** [comment]
@brainfunked review.

File: base.rb [hunk 1]
```diff
@@ -5,6 +5,10 @@ class Base < Sinatra::Base
 
   set :environment, ENV['RACK_ENV'] || 'development'
 
+  set :logging, true
+
+  set :logging, Logger::INFO
```
Comment (shtripat) [inline base.rb]: Should we hard code log level to INFO always here?

File: base.rb [hunk 2]
```diff
@@ -72,8 +76,37 @@ class Base < Sinatra::Base
 
   protected
 
+  def monitoring
+    yaml = etcd.get('/monitoring/config').value
+    config = YAML.load(yaml)
+    @monitoring = Tendrl::MonitoringApi.new(config)
+  rescue Etcd::KeyNotFound
+    logger.debug 'Monitoring API not enabled.'
```
Comment (brainfunked) [inline base.rb]: info seems appropriate here. Administrator would need to know that the API hasn't detected the monitoring API.

**Comment by anivargi** [comment]
@brainfunked please review asap.

**Pull request merged**